src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -156,6 +183,14 @@ function useCheckEncryptionState(state: State, setState: (state: State) => void)
         if (state === "loading") checkEncryptionState();
     }, [checkEncryptionState, state]);
 
+    useTypedEventEmitter(matrixClient, ClientEvent.AccountData, (event: MatrixEvent): void => {
+        const type = event.getType();
+        // Recheck the status if this account data has been updated as this implies it has changed
+        if (type === "m.org.matrix.custom.backup_disabled") {
+            checkEncryptionState();
+        }
+    });
```

Comment:
Moving from https://github.com/element-hq/element-web/pull/29113/files#r1956092346

Mostly because it's easy to react to account data being updated, whereas there are no events fired for the other things changing.

Reply:
ok, it might be good to add a comment saying that we're deliberately doing this.

Reply:
I was thinking something along the lines of:

```
// Note that there are other things that could potentially invalidate the state of this tab (such as the
// backup being deleted from another client), but that is hard to monitor for, so we don't bother.
```

That said, this whole thread might be obsolete.